app/src/main/java/org/mozilla/fenix/customtabs/CustomTabsIntegration.kt

```diff
@@ -63,18 +78,6 @@ class CustomTabsIntegration(
                     DisplayToolbar.Indicators.SECURITY
                 )
             }
-
-            toolbar.display.icons = toolbar.display.icons.copy(
-                trackingProtectionTrackersBlocked = lottieDrawable,
-                trackingProtectionNothingBlocked = AppCompatResources.getDrawable(
-                    activity,
-                    R.drawable.ic_tracking_protection_enabled
-                )!!,
```

Comment:
The code was previously split into three chunks, 1) create drawable, 2) set indicators, 3) set drawable.  I've moved chunk 3 to live with 1.